spec/stat_tracker_spec.rb

```diff
@@ -25,6 +25,18 @@
     expect(@stat_tracker).to be_a(StatTracker)
   end
 
+  it "can create games in an array" do
+    expect(@stat_tracker.create_games("./data/games_sampl.csv")).to be_a Array
+  end
+
+  it "can create teams in an array" do
+    expect(@stat_tracker.create_teams("./data/teams_sampl.csv")).to be_a Array
+  end
+
+  it "can create game teams in an array" do
+    expect(@stat_tracker.create_game_teams("./data/game_teams_sampl.csv")).to be_a Array
+  end
```

Comment:
Should the above three also test that the array is full of Game objects, Team objects, and Game Team objects respectively?

Reply:
For sure!
I'll add those changes now!